src/scheduler.rs

```diff
@@ -513,6 +514,7 @@ async fn inbox_fetch_idle(ctx: &Context, imap: &mut Imap, mut session: Session)
         }
     };
 
+    maybe_send_statistics(ctx).await.log_err(ctx).ok();
```

Comment:
Another option is to do this before `send_smtp_messages()` in `smtp_loop()` so that it doesn't wake up an extra time. I don't think this requires optimization though, just trying to find a more natural place for it